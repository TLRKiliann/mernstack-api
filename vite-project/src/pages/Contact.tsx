import { Component } from 'react'
import '../stylePages/Contact.scss'

export default class Contact extends Component {
  render() {
    return(
      <div>
        <div className="title--contact">
          <h1>Contact</h1>
        </div>
      </div>
    )
  }
}