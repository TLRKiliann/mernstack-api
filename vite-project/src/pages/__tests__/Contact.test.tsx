import React from "react";
import {assert, assertType, expectTypeOf, beforeEach, afterEach, describe, expect, test, it, vi} from 'vitest';
import {fireEvent, render, screen} from '@testing-library/react';
//import "@testing-library/jest-dom/extend-expect"
//import "@testing-library/jest-dom";
//import { act } from 'react-dom/test-utils';
import {create} from 'react-test-renderer';
import Contact from "../Contact.tsx";

test('MatchSnapShot test Contact', () => {
	const tree = create(<Contact />)
	expect(tree.toJSON()).toMatchSnapshot();
})