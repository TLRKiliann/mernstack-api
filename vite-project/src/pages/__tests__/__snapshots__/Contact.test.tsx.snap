// Vitest Snapshot v1

exports[`MatchSnapShot test Contact 1`] = `
<div>
  <div
    className="title--contact"
  >
    <h1>
      Contact
    </h1>
  </div>
</div>
`;
